assignment 6/day 7/index.js

```diff
@@ -0,0 +1,29 @@
+const fs = require('fs');
+
+function readFile(file) {
+  return new Promise(function(resolve, reject) {
+    fs.readFile(file, 'utf8', function(err, data) {
+      if (err) return reject(err);
+      resolve(data);
+    })
+  });
+}
+
+readFile("./files/mongodb.txt").then(function(data) {
+  console.log(data);
+  return readFile("./files/expressjs.txt");
+}).then(function(data) {
```

Comment:
the indentation in .then chaining has a specific manner. An example is shown below:

```
someAsyncTask(param)
 .then((result) => {
    console.log('do something');
    return otherAsyncTask(result);
  })
 .then((result2) => {
    console.log('do something else');
    return thirdAsyncTask(result2);
  })
 .catch((error) => {
    console.log('something went bad');
    console.log(err)
  });
```